src/assets/scripts/client/constants/colors/terrainThemes.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * @property TERRAIN_COLOR_DEFAULT
+ * @type {Object}
+ * @final
+ */
+const TERRAIN_COLOR_DEFAULT = {
```

Comment:
Do you mean just the `BORDER_OPACITY` and `FILL_OPACITY`? the theme itself is probably wise to keep separate, to make it easy to add different terrain coloring themes and treat them as a unit that belongs to an overall "theme" used by the canvas.